tensorflow/tools/ci_build/builds/docker_test.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
```

Comment:
What is the goal of this? Are we trying to automate the release process? Or are we trying to run python tests for every build (master, pull requests, ...) in better isolation?

For the better isolation it would be better (simpler, multiplatform, easier and faster) to use virtualenv. We can make release more automated later.